src/Workspaces/CSharp/Portable/Extensions/ExpressionSyntaxExtensions.cs

```diff
@@ -2621,6 +2623,9 @@ public static OperatorPrecedence GetOperatorPrecedence(this ExpressionSyntax exp
                 case SyntaxKind.CheckedExpression:
                 case SyntaxKind.UncheckedExpression:
                 case SyntaxKind.AnonymousMethodExpression:
+                // unsafe code
```

Comment:
❓ Is this part of an unrelated change?

Reply:
No.  These were part of hte change where i check if it's ok to remove parens and i need to get operator precedence to do so.  We were missing operator precedence for unsafe operators (like ```*x```), which made us not know properly if we could, for example, remove parens in ```(void*)(*x)```